lib/datasource/git-tags/index.js

```diff
@@ -0,0 +1,22 @@
+const { getRemoteInfo } = require('isomorphic-git');
+
+async function getPkgReleases({ lookupName }) {
+  try {
+    const info = await getRemoteInfo({
+      url: lookupName,
+    });
+    if (info && info.refs && info.refs.tags) {
+      const tags = Object.keys(info.refs.tags);
+      return tags
+        .map(tag => tag.replace(/^v(?=[0-9])/, ''))
+        .filter(tag => !/\^/.test(tag));
```

Comment:
Can you document what the intention of this replacing/filtering is?

Please also model is on the existing github tags fetching we have. i.e. do not filter more than than `github.getPkgReleases` already does

Reply:
I've checked the filtering is correct on my setup, but it's problematic to make such test since it should make real requests.